airtable/emails.js

```diff
@@ -222,13 +202,17 @@ let enqueueButton = await input.buttonsAsync(
     ],
 );
 
-const messagesToCreate = await computeMessagesToCreate();
+let messagesToCreate = await computeMessagesToCreate();
 
-if (messagesToCreate.length > 0) {
-    messagesTable.createRecordsAsync(messagesToCreate);
-    output.markdown(`**Success!** Enqueued ${messagesToCreate.length} new messages`)
-} else {
+if (messagesToCreate.length == 0) {
     output.markdown(`No new messages to enqueue.`);
+} else {
+    if (messagesToCreate.length > 50) {
+        output.markdown(`You can only create 50 emails at once, but we found ${messagesToCreate.length} Doing the first 50 now... remember to do this again to get the rest!`)
```

Comment:
ooh fun. is this an airtable limitation?

Reply:
haha yup, testing on staging somehow it got to 118 emails it wanted to send >.<

Reply:
heh nice. would be awesome if we get to real cities creating 50 emails at a time :)